kolibri/utils/options.py

```diff
@@ -304,135 +257,207 @@ def origin_or_port(value):
         },
     },
     "Database": {
         "DATABASE_ENGINE": {
             "type": "option",
             "options": ("sqlite", "postgres"),
             "default": "sqlite",
-            "envvars": ("KOLIBRI_DATABASE_ENGINE",),
+            "description": "Which database backend to use, choices are sqlite or postgresql",
+        },
+        "DATABASE_NAME": {
+            "type": "string",
+            "description": """
+                For SQLite - the name of a database file to use for the main Kolibri database.
+                For Postgresql, the name of the database to use for all Kolibri data.
+            """,
         },
-        "DATABASE_NAME": {"type": "string", "envvars": ("KOLIBRI_DATABASE_NAME",)},
         "DATABASE_PASSWORD": {
             "type": "string",
-            "envvars": ("KOLIBRI_DATABASE_PASSWORD",),
+            "description": "The password to authenticate with when connecting to the database, Postgresql only.",
+        },
+        "DATABASE_USER": {
+            "type": "string",
+            "description": "The user to authenticate with when connecting to the database, Postgresql only.",
+        },
+        "DATABASE_HOST": {
+            "type": "string",
+            "description": "The host on which to connect to the database, Postgresql only.",
+        },
+        "DATABASE_PORT": {
+            "type": "string",
+            "description": "The port on which to connect to the database, Postgresql only.",
         },
-        "DATABASE_USER": {"type": "string", "envvars": ("KOLIBRI_DATABASE_USER",)},
-        "DATABASE_HOST": {"type": "string", "envvars": ("KOLIBRI_DATABASE_HOST",)},
-        "DATABASE_PORT": {"type": "string", "envvars": ("KOLIBRI_DATABASE_PORT",)},
     },
     "Server": {
         "CHERRYPY_START": {
             "type": "boolean",
             "default": True,
-            "envvars": ("KOLIBRI_CHERRYPY_START",),
+            "description": "DEPRECATED - do not use this option, use the 'kolibri services' command instead.",
         },
         "CHERRYPY_THREAD_POOL": {
             "type": "integer",
             "default": calculate_thread_pool(),
-            "envvars": ("KOLIBRI_CHERRYPY_THREAD_POOL",),
+            "description": "How many threads the Kolibri server should use to serve requests",
         },
         "CHERRYPY_SOCKET_TIMEOUT": {
             "type": "integer",
             "default": 10,
-            "envvars": ("KOLIBRI_CHERRYPY_SOCKET_TIMEOUT",),
+            "description": """
+                How long a socket should wait for data flow to resume before
+                it considers that the connection has been interrupted.
+                Increasing this may help in situations where there is high latency on a network,
+                or the bandwidth is bursty, and it is expected that data flow may be interrupted
+                but not indicative of the connection failing.
```

Comment:
Unclear around what `is not indicative`...? 😕

Reply:
The interruption of data flow is not indicative of the connection failing - thoughts on how to rephrase better?

Reply:
Maybe: `Increasing this may help in situations where there is high latency on a network or the bandwidth is bursty, with some expected data flow interruptions which may not be indicative of the connection failing.`